Dockerfile

```diff
@@ -103,6 +121,7 @@ EXPOSE 7681
 WORKDIR /home/user
 
 COPY entry.sh /bin
+RUN echo 'set-option -g status off' >> ~/.tmux.conf
```

Comment:
This makes it so tmux won't have a status line at the bottom of the screen. Since we're using tmux just to have the shell be persistent, that'd just be in the way I think.